src/main/resources/api/service_openapi.yaml

```diff
@@ -459,6 +459,66 @@ components:
         hybridConnectionName:
           type: string
           description: The user-provided name to identify the Relay HybridConnection.
+    AzureContainerInstance:
+      type: object
+      required:
+        - resourceGroup
+        - containerGroupName
+        - containers
+        - operatingSystemTypes
+      properties:
+        resourceGroup:
+          $ref: '#/components/schemas/AzureResourceGroup'
+        containerGroupName:
+          type: string
+          description: The name of the container group
+        containers:
+          type: array
+          items:
+            type: object
+            required:
+              - name
+              - image
+            properties:
```

Comment:
we need to expose network options as well. also for image, we'll need to use ACR from a different subscription, so we'll need to pass username and password as well.

Or do you know of a way to make ACR public?

Reply:
Add credential for registry and container ip address.